public/index.html

```diff
@@ -36,12 +36,13 @@
     <script src="https://cdn.fromdoppler.com/loader/v1/loader.js"></script>
     <!-- End AssetServices -->
   </head>
-  <body>
+  <body class="dp-library">
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div class="dp-library dp-wrapper">
+    <div id="modal-wrapper" class="dp-wrapper">
       <header id="root-header"></header>
       <div id="root"></div>
       <footer id="root-footer"></footer>
+      <div id="root-modal"></div>
```

Comment:
If I understand it well, it is a breaking change because it will require changes in all the places where the Editors WebApp is used.

Is avoiding them possible?

Reply:
Yes that was posible using the _react-modal_ props `portalClassName `and [`ariaHideApp`](https://github.com/FromDoppler/doppler-editors-webapp/pull/398#discussion_r1087181696)

Reply:
[Let's chat](https://makingsense.slack.com/archives/D023SNMD6E7/p1674735588398049) about it.